### DISCUSSION TITLE
Create 1x2 split panes with command running in one pane but not the rest

### DISCUSSION THREAD

**Discussion by NovaViper** [wezterm/wezterm#3681] [Q&A] [answered]
Hey, I'm trying to figure out how to create a startup function that makes the terminal with a main pane, and 2 split panes like in the picture
![image](https://user-images.githubusercontent.com/7191115/236317048-85e11c7c-9034-4cf9-b307-d0e825986797.png)

But with one added function, have that third pane be the only terminal to run a command (like cmatirx as shown in the screenshot). So far I got the splits to appear properly but I can't figure out how to make one of the 3 panels run a command while leaving the rest to use either a command from when the computer opens a program or just the default shell.
This is what I have so far:
```
wezterm.on('gui-startup', function(cmd)
  local tab, pane, window = mux.spawn_window(cmd or {})
  -- Create a right side pane
  local right_pane = pane:split { direction = 'Right', size = 0.4 }
  -- Split right pane into two, with new pane on bottom
  local bottom_pane = right_pane:split { direction = 'Bottom'}
  -- Activate primary left pame
  pane:activate()
end)
```

**Comment by wez** [comment, marked as answer]
I think you want:

```lua
local bottom_pane = right_pane:split { direction = 'Bottom', args = {"some", "program"}}
```

**Reply by NovaViper** [reply]
Hey I know this is late but I actually came across the post again in configuring Wezterm again and it worked! Thank you and sorry for not replying back in a timely manner!
